godot-core/src/builtin/signal.rs

```diff
@@ -213,3 +216,310 @@ impl fmt::Display for Signal {
         write!(f, "{}", self.to_variant())
     }
 }
+
+// ----------------------------------------------------------------------------------------------------------------------------------------
+// Implementation of a rust future for Godot Signals
+#[cfg(since_api = "4.2")]
```

Comment:
Uppercase "Rust" + empty line after:
```suggestion
// ----------------------------------------------------------------------------------------------------------------------------------------
// Implementation of a Rust future for Godot signals

#[cfg(since_api = "4.2")]
```

(might not be needed with the next change)